example-server/app/public/index.html

```diff
@@ -1,108 +1,120 @@
+          <input type="submit" ng-click="send(event)" value="Send"/>
+        </form>
+        <spen ng-show="error.length > 0" class="text-danger">{{error}}</spen>
+        <pre ng-show="sent" class="text-success">Sent {{event | json}}</pre>
+      </div>
+      <div ng-controller="ViewCartController" ng-show="selectedEvent ==='viewCart'">
+        <form novalidate class="simple-form">
+          <H2>View Cart</H2>
+          <textarea ng-model="eventString" rows="20" cols="50"></textarea>
+          </br>
+          <input type="submit" ng-click="send(event)" value="Send"/>
+        </form>
+        <spen ng-show="error.length > 0" class="text-danger">{{error}}</spen>
+        <pre ng-show="sent" class="text-success">Sent {{event | json}}</pre>
+      </div>
+      <div ng-controller="RemoveFromCartController" ng-show="selectedEvent ==='removeFromCart'">
+        <form novalidate class="simple-form">
+          <H2>Remove From Cart</H2>
+          <textarea ng-model="eventString" rows="20" cols="50"></textarea>
+          </br>
+          <input type="submit" ng-click="send(event)" value="Send"/>
+        </form>
+        <spen ng-show="error.length > 0" class="text-danger">{{error}}</spen>
+        <pre ng-show="sent" class="text-success">Sent {{event | json}}</pre>
+      </div>
+      <div ng-controller="OrderController" ng-show="selectedEvent ==='order'">
+        <form novalidate class="simple-form">
+          <H2>Order</H2>
+          <textarea ng-model="eventString" rows="20" cols="50"></textarea>
+          </br>
+          <input type="submit" ng-click="send(event)" value="Send"/>
+        </form>
+        <spen ng-show="error.length > 0" class="text-danger">{{error}}</spen>
+        <pre ng-show="sent" class="text-success">Sent {{event | json}}</pre>
+      </div>
+      <div ng-controller="AutoSearchController" ng-show="selectedEvent ==='autoSearch'">
+        <form novalidate class="simple-form">
+          <H2>Auto Search</H2>
+          <textarea ng-model="eventString" rows="20" cols="50"></textarea>
+          </br>
+          <div>
+            Search body: <input type="text" ng-model="searchBody" size="50"/>
+            <span ng-show="jsonError" style="color:red">Invalid JSON!</span>
+            <br/><br/>
+          </div>
+          <input type="button" ng-click="overrideSearch()" value="Override Search Endpoint"/>
+          <input ng-show="allowSearchOverride" type="text" ng-model="searchPath" size="100"/><br/>
+          <input type="button" ng-click="overrideBeacon()" value="Override Beacon Endpoint"/>
+          <input ng-show="allowBeaconOverride" type="text" ng-model="beaconPath" ng-change="beaconPathChange()" size="100"/><br/>
+          <input type="button" ng-click="overrideApiKey()" value="Override API Key"/>
+          <input ng-show="allowOverrideApiKey" type="text" ng-model="apiKey" size="100"/><br/>
+          Manually Send Beacon&nbsp&nbsp&nbsp<input type="checkbox" ng-model="manualBeacon"/><br/>
```

Comment:
I don't think we need this checkbox. 

If they supply the direct beacon path, it's assumed that it's manual, right? And if they don't, it can't be manual.